get-pip.py

```diff
@@ -27,6 +27,14 @@
 import struct
 import tempfile
 
+# Check for wrong embedded python settings
+if 'site' not in sys.modules:
+    print('site package is not loaded by default. Please edit python%d%d._pth' % (sys.version_info[0], sys.version_info[1]))
```

Comment:
It's better to write this to stderr, rather than stdout:
```python
    print(
        'site package is not loaded by default. Please edit python%d%d._pth'
        % sys.version_info[:2],
        file=sys.stderr
    )
```

Reply:
resolved in pull https://github.com/pypa/get-pip/pull/32